frontend/src/components/auth/WithAuth.js

```diff
@@ -32,14 +33,15 @@ const withAuth = WrappedComponent => {
         this.props.history.push('/register')
       }
     }
+
     render() {
       return (
         <div>
           {this.state.verified ? (
             <WrappedComponent {...this.props} verified={this.state.verified} />
           ) : (
             <div>
-              <p> You are not authenticated </p>
+              <p>You are not authenticated.</p>
```

Comment:
No need to translate this I would imagine; this will eventually be removed with a reroute to the login page.

Reply:
It's less about needing to translate but the fact that the wrapper component fucks with how I'm doing things